clickhouse/templates/ingress-tabix.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 spec:
   rules:
-    host: {{ .Values.tabix.ingress.host }}
+  - host: {{ .Values.tabix.ingress.host }}
```

Comment:
I think you need to apply the same thing to hosts in tls ? 

Cf https://github.com/sentry-kubernetes/charts/pull/219/files

Reply:
you are correct, I didn't notice that.

Reply:
done